util/gh-pages/index.html

```diff
@@ -25,7 +25,62 @@
         blockquote { font-size: 1em; }
         [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak { display: none !important; }
 
-        .form-inline .checkbox { margin-right: 0.6em }
+        .dropdown-menu {
+            color: var(--fg);
+            background: var(--theme-popup-bg);
+            border: 1px solid var(--theme-popup-border);
+        }
+
+        .dropdown-menu .divider {
+            background-color: var(--theme-popup-border);
+        }
+
+        .dropdown-menu .checkbox {
+            width: 100%;
+            display: block;
+            clear: both;
+            font-weight: 400;
+            line-height: 1.42857143;
+            white-space: nowrap;
+            margin: 0;
+         }
+
+         .dropdown-menu .checkbox label {
+            padding: 3px 20px;
+            width: 100%;
+         }
+
+         .dropdown-menu .checkbox input {
+             position: relative;
+             margin: 0 0.5rem 0;
+             padding: 0;
+             vertical-align: middle;
+         }
+
+         .dropdown-menu .checkbox:hover {
+            background-color: var(--theme-hover);
+        }
+
+        div.panel div.panel-body button.dropdown-toggle {
+            background: var(--searchbar-bg);
+            color: var(--searchbar-fg);
+            border-color: var(--theme-popup-border);
+        }
+
+        div.panel div.panel-body button.dropdown-toggle:hover {
+            box-shadow: 0 0 3px var(--searchbar-shadow-color);
+        }
+
+        div.panel div.panel-body .open button.dropdown-toggle {
+            background: var(--searchbar-bg);
+            color: var(--searchbar-fg);
+            border-color: var(--theme-popup-border);
+            filter: brightness(90%);
+        }
```

Comment:
Bootstrap theming is normally done by downloading `bootstrap-theme.css` and modifying the values. We don't need an entire theme, but I think we should copy parts of that code, and add a comment saying as much.

Reply:
These variables are coming from rust mdBook styles. Which code do you mean?